src/Elastic.Apm/Api/CapturedException.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+using Elastic.Apm.Model.Payload;
+
+namespace Elastic.Apm.Api
+{
+	public class CapturedException
```

Comment:
Is the reason for the confusion that there is no public method that you can pass an instance of this? Because if it is, then you are right :) There is not method that accepts this as an argument. I think I made it public, because there was a field of this on `IError`, and that field was added to `IError`, because it's required (or to be precise either this, or another field called `Log`, which in .NET isn't that useful, so we only have `Exception`). But that's probably not nice, since we expose something that cannot be used. Exceptions should be captured with the `CaptureException` and `CaptureError` methods on `ISpan`, `ITracer`,  or `ITransaction`. I made this class private.